frontend/packages/dev-console/src/components/pipelines/pipeline-form/SecretsList.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { ResourceLink, Firehose, FirehoseResult } from '@console/internal/components/utils';
+import { SecretModel } from '@console/internal/models';
+import { SecretType } from '@console/internal/components/secrets/create-secret';
+import { SecondaryStatus } from '@console/shared';
+import './SecretsList.scss';
+
+type SecretsProps = {
+  secrets?: FirehoseResult<any[]>;
```

Comment:
`SecretKind` can be added instead of `any`

Reply:
Done.

Reply:
Done.